openapiGenerator.go

```diff
@@ -681,6 +694,12 @@ func (g *openapiGenerator) parseComments(desc protomodel.CoreDesc) (comments str
 			if shouldNotRenderDesc(l) {
 				continue
 			}
+
+			// If this is an ignored kube marker, we'll skip this line, preventing validation and rendering marker in output
+			if strings.HasPrefix(l, markers.Kubebuilder) && isIgnoredKubeMarker(ignoredKubeMarkersRegexp, l) {
```

Comment:
I'd move `isIgnoredKubeMarker(ignoredKubeMarkersRegexp, l)` within the conditional on L703 so that all marker conditionals are grouped together.